shared/contracts/BaseTemplate.sol

```diff
@@ -65,15 +65,28 @@ contract BaseTemplate is APMNamehash, IsContract {
         acl.createPermission(address(this), app, permission, address(this));
     }
 
+    function _createPermissions(ACL acl, address[] grantees, address app, bytes32 permission, address manager) internal {
```

Comment:
Perhaps it would be nice to make this function more explicit, like `_createPermissionsAndSetManager`